_posts/2023-10-09-towards_3.0.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Towards MDAnalysis 3.0
+---
+
+## Overview
+
+MDAnalysis is beginning to plan for our 3.0 release! Following discussions at our first [UGM][], we (the MDAnalysis [Core Developers][]) have decided on a set priorities for our this exciting milestone. Our primary goals can be divided into several main themes.
+
+
+### A trimmed down core library
+
+As MDAnalysis continues to grow along with the field of molecular simulation, we are aiming to continue to provide a highly functional, easily maintainable package that can be the foundation of a wide variety of downstream tools.
+
+Following the successful introduction of the [MDAKit ecosystem][] (modelled on the SciKit ecosystem, see the [MDAKits whitepaper][]) as part of our [EOSS4][] work, which provides infrastructure for creating FAIR packages that address specific scientific questions in the analysis of molecular simulation, we are aiming to move some more specialized analyses currently present in the MDAnalysis core library to MDAKits. Several analysis modules, including  hole2, encore, path-similarity, helanal and waterdynamics are being assessed to undergo this transition. Indeed some of these packages are already deployed as MDAKits, and are deprecated in the core library.
+
+This change will allow for a reduction in the core dependencies of MDAnalysis and reduce the maintenance burden of specialist analyses in the core library. This will also empower those developing specialist analysis code to rapidly develop and prototype their code in well supported downstream packages. Additionally, core developers will be able to spend more cycles on core competencies, fixing critical issues and planning future improvements, ensuring that MDAnalysis can be easily and effectively maintained for years to come.
+
+### Enhancements
+
+We have identified several areas that we would like to target for future improvement of MDAnalysis across three key themes.
+
+* Improving interoperability
+    - ....
+
+* Support for streaming and cloud storage
+    - ....
+
+* Enhanced parallelism and performance
+    - ...
+
```

Comment:
It would be great to tie EOSS 6 to this roadmap yes -- could we please add guessers/chemical contexts in here? While they didn't come up in the UGM planning meeting, we did get feedback during the 3.0 panel asking for something similar. And if we keep encouraging MDAKits and more diverse analyses, they'll become more and more useful IMO.

Reply:
I think I initially grouped them as "scientifically aware operations" and lumped in guessers, cheminformatics, and units, but since we're not properly doing units that may not be the best name.

Reply:
@lilyminium I have tried to link with EOSS6 proposal now. Have a look and see if you like it.